infrastructure/environments/dplplat01/configuration/ingress-nginx/ingres-nginx-values.template.yaml

```
@@ -16,8 +16,12 @@ controller:
   replicaCount: 2
   autoscaling:
     enabled: true
-    minReplicas: 2
-    maxReplicas: 10
+    minReplicas: 5
+    maxReplicas: 15
+  resources:
+    requests:
+      cpu: 1000m
```

I think this is sufficient for CPU
```suggestion
      cpu: 250m
```